packages/dashboard/examples/single-state-filters.json

```diff
@@ -0,0 +1,390 @@
+{
```

Comment:
could this go in storybook or should it be attached to the ticket? otherwise delete